hazelcast/src/main/java/com/hazelcast/jet/impl/JobAndSqlSummary.java

```diff
@@ -41,7 +43,8 @@ public JobAndSqlSummary(
             long submissionTime,
             long completionTime,
             String failureText,
-            SqlSummary sqlSummary) {
+            SqlSummary sqlSummary,
+            @Nullable String suspensionCause) {
```

Comment:
Just a question: With this approach, we wouldn't know if the `suspensionCause` is null because it was sent by an old protocol (basically `isSuspensionCauseExists` is false) or if it is sent by a new protocol and it is actually null. Do we need to distinguish these cases?

Reply:
I wouldn't distinguish them, because it is fine to have it null with new protocol (e.g., light job doesn't support suspensionCause)